content/en/blog/plugins-early-access.md

```diff
@@ -0,0 +1,157 @@
+
+We do plan on relaxing these restrictions to allow running multiple source plugins at once, once we tackle the concurrency, fairness, and resource utilization issues that result from having multiple streams of events being processed by a single falco rules engine.
+
+### Cloudtrail Plugin
+
+Along with the introduction of plugins support, we've written a source plugin [cloudtrail](https://github.com/falcosecurity/plugins/tree/master/plugins/cloudtrail) that can read cloudtrail logs and pass them to the rules engine as events.
+
+The plugin can be configured to obtain log files in a number of ways, depending on the `open_params` value in `falco.yaml`:
+
+* A [S3](https://aws.amazon.com/s3/) bucket: `s3://<S3 Bucket Name>[/<Optional Prefix>]`.
+* A [SQS](https://aws.amazon.com/sqs/) queue that passes along [SNS notifications](https://docs.aws.amazon.com/awscloudtrail/latest/userguide/configure-sns-notifications-for-cloudtrail.html) about new log files: `sqs://<SQS Queue Name>`
+* A local filesystem path: `Any path`.
+
+The plugin also exports fields that extract information from a cloudtrail event, such as the event time, the aws region, S3 bucket/EC2 instance name, etc.
+
+The plugin uses the same authentication mechanisms used by the [AWS Go SDK](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials):
+
+* **Environment Variables**: specify the aws region with `AWS_REGION=xxx`, the access key id with `AWS_ACCESS_KEY_ID=xxx`, and the secret key with `AWS_SECRET_ACCESS_KEY=xxx`.
+* **Shared Configuration Files**: specify the aws region in a file at `$HOME/.aws/config` and the credentials in a file at `$HOME/.aws/credentials`.
+
+#### New Rules for AWS Cloudtrail
+
+Along with the plugin, we've created new Falco rules that look for suspicious/notable/interesting activity in cloudtrail logs. Here is an example rule "Console Login Without MFA" (Multi Factor Authentication):
+
+```yaml
+- rule: Console Login Without MFA
+  desc: Detect a console login without MFA.
+  condition:
+    ct.name="ConsoleLogin" and ct.error=""
+    and json.value[/userIdentity/type]!="AssumedRole" and json.value[/responseElements/ConsoleLogin]="Success"
+    and json.value[/additionalEventData/MFAUsed]="No"
+  output:
+    Detected a console login without MFA (requesting user=%ct.user, requesting IP=%ct.srcip, AWS region=%ct.region)
+  priority: CRITICAL
+  source: aws_cloudtrail
+```
+
+The rule source is `aws_cloudtrail`. When falco is run with a loaded cloudtrail plugin, it will evaluate these rules because the plugin's event source matches the source in the rule. The fields `ct.name`, `ct.error`, `ct.srcip`, etc are all resolved by the plugin via a call to the plugin's extract fields function. For example, when `ct.name` is "ConsoleLogin", that represents a console login attempt. The full set of fields supported by the cloudtrail plugin are in the [README](https://github.com/falcosecurity/plugins/blob/master/plugins/cloudtrail/README.md).
```

Comment:
The rule source is `aws_cloudtrail`. When Falco is run with a loaded Cloudtrail plugin, it will evaluate these rules because the plugin event source matches the source in the rule. The fields `ct.name`, `ct.error`, `ct.srcip`, etc are resolved by the plugin via a call to the plugin's extract fields function. For example, matching `ct.name`  with "ConsoleLogin" indicates a console login attempt. The full set of fields supported by the Cloudtrail plugin are in the [README](https://github.com/falcosecurity/plugins/blob/master/plugins/cloudtrail/README.md).